<hibernate-mapping xmlns="urn:nhibernate-mapping-2.2"
assembly="HRR.Core"
namespace="HRR.Core.Domain">
    <class name="ApplicationException" table="ApplicationException">
        <id name="ID" column="ID">
            <generator class="identity"></generator>
        </id>
        <property name="AccountID" />
        <property name="CurrentURL" />
        <property name="StackTrace" />
        <property name="Message" />
        <property name="InnerMessage" />
        <property name="ExceptionDate" />
        <property name="UserID" />
        <property name="SessionData" />
        <property name="ExceptionType" />
        <many-to-one name="UserRef" class="HRR.Core.Domain.Person, HRR.Core" column="UserID" insert="false" update="false" cascade="none" />
        <many-to-one name="AccountRef" class="HRR.Core.Domain.Account, HRR.Core" column="AccountID" insert="false" update="false" cascade="none" />
    </class>
</hibernate-mapping>